install.js

```diff
@@ -29,6 +29,7 @@ download({
   platform: process.env.npm_config_platform,
   arch: process.env.npm_config_arch,
   strictSSL: process.env.npm_config_strict_ssl === 'true',
+  force: process.env.force_no_cache || false,
```

Comment:
the environment variable will always be a string. You might want to coerce the input to a boolean: `Boolean(process.env.force_no_cache)`.


```
❯ node
> Boolean(null)
false
> Boolean(undefined)
false
> Boolean('')
false
> Boolean('true')
true
> Boolean('false')
true
> Boolean('1')
true
```